src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -1091,8 +1091,8 @@
   </Event>
 
   <Event name="GCLocker" category="Java Virtual Machine, GC, Detailed" description="GC Locker Information" label="GC Locker" startTime="true" thread="true" stackTrace="true">
```

Comment:
"GC Locker Information" is not very useful. Remove the description completely or provide more information.

Reply:
removed.